app/app.go

```diff
@@ -412,6 +414,7 @@ func NewBabylonApp(
 		// Integration related modules
 		bsctypes.ModuleName,
 		zctypes.ModuleName,
+		btcstkconsumertypes.ModuleName,
```

Comment:
:thinking: we didn't had it setup?

Reply:
Hmm I think this is because the `btcstkconsumer` module does not have any logic in Begin/EndBlocker. But doesn't bother adding them here